extensions/vertx/runtime/src/main/java/io/quarkus/vertx/ConsumeEvent.java

```diff
@@ -83,12 +83,19 @@
      */
     boolean blocking() default false;
 
+    /**
+     * @return {@code true} if the <em>blocking</em> consumption of the event must be ordered, meaning that the method
+     *         won't be called concurrently. Instead it serializes all the invocations based on the event order.
+     *         {@code ordered} must be used in conjunction with {@code blocking=true} or {@code @Blocking}.
```

Comment:
I don't want to be picking but something like `{@link io.smallrye.common.annotation.Blocking}` would be probably better ;-)